From 4dddf393d6569b705c87dff092dee34439059d4b Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Mon, 4 Mar 2019 14:04:53 +0000
Subject: [PATCH] MANTA-4140 manta repositories should not use git+ssh:// URIs
 in package.json files Reviewed by: Kelly McLaughlin
 <kelly.mclaughlin@joyent.com>

---
 .gitmodules            | 2 +-
 bin/generate-config.js | 8 +++++---
 package.json           | 5 +++--
 3 files changed, 9 insertions(+), 6 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index 56a38f3..45499d1 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,6 +1,6 @@
 [submodule "deps/manta-scripts"]
 	path = deps/manta-scripts
-	url = git@github.com:joyent/manta-scripts.git
+	url = git://github.com/joyent/manta-scripts.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
 	url = git://github.com/davepacheco/javascriptlint.git
diff --git a/bin/generate-config.js b/bin/generate-config.js
index 8573689..9591fe1 100755
--- a/bin/generate-config.js
+++ b/bin/generate-config.js
@@ -7,13 +7,14 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 var bunyan = require('bunyan');
 var fs = require('fs');
 var path = require('path');
 var sdc = require('sdc-clients');
+var ufds_mod = require('ufds');
 var vasync = require('vasync');
 
 var LOG = bunyan.createLogger({
@@ -79,7 +80,7 @@ function setupSingleDcClients(_, cb) {
 					'ufdsConfig': self.UFDS_CONFIG
 				}, 'connecting to ufds');
 
-				self['UFDS'] = new sdc.UFDS(self.UFDS_CONFIG);
+				self['UFDS'] = new ufds_mod(self.UFDS_CONFIG);
 
 				self['UFDS'].on('ready', function (err) {
 					self.log.debug({
@@ -99,7 +100,8 @@ function setupSingleDcClients(_, cb) {
 				self['SAPI'] = new sdc.SAPI({
 					log: self.log,
 					url: url,
-					agent: false
+					agent: false,
+					version: '*'
 				});
 				subcb();
 			}
diff --git a/package.json b/package.json
index 26f6a6a..47170a6 100644
--- a/package.json
+++ b/package.json
@@ -1,13 +1,14 @@
 {
 	"name": "marlin-dashboard",
-	"version": "0.1.0",
+	"version": "0.1.2",
 	"description": "marlin dashboard server",
 	"private": true,
 	"dependencies": {
 		"bunyan": "0.23.0",
 		"jsprim": "0.6.1",
 		"kang": "1.0.1",
-		"sdc-clients": "git+ssh://git@github.com:joyent/node-sdc-clients.git#c962959",
+		"sdc-clients": "12.0.0",
+		"ufds": "1.5.0",
 		"vasync": "1.5.0"
 	},
 	"license": "MPL-2.0"
-- 
2.21.0

